### DISCUSSION TITLE
I apologise

### DISCUSSION THREAD

**Discussion by KG240202** [community/community#60343] [Other Feature Feedback, Questions, & Ideas] [answered]
@nefarius

General

### Body

I wanted to apologise to user and programmer Nefarius and other users for my offensive and vulgar behaviour on the DSHidMini repository a few days ago. I was frustrated with getting a Dualshock 3 controller to work on PC. But I appreciate these programs are well-developed and cleverly executed by talented professionals like yourself, who made gamepad emulation possible with more controllers other than Xbox. 
I will follow guides on external sites and build the solution myself to work it. And thank your for your software. :)

**Comment by gooddavvy** [comment, marked as answer]
Hi @KG240202, 

You may have been permabanned, I've had a similar situation in another repo. @Github may accept our apologies but not unban us.

**Reply by KG240202** [reply]
I'd be glad to have my apology heard, raher than being unbanned. But thank you.

**Reply by nefarius** [reply]
Accepted and unbanned, welcome back 😉

**Reply by gooddavvy** [reply]
@KG240202 You're welcome! Oh and please mark that comment as the answer!

**Reply by gooddavvy** [reply]
@KG240202 please mark that comment as the answer!
